src/lib.rs

```diff
@@ -496,3 +496,29 @@ impl<'ui> Ui<'ui> {
     pub fn menu<'p>(&self, label: ImStr<'p>) -> Menu<'ui, 'p> { Menu::new(label) }
     pub fn menu_item<'p>(&self, label: ImStr<'p>) -> MenuItem<'ui, 'p> { MenuItem::new(label) }
 }
+
+//Widgets: Combos 
+impl<'ui> Ui<'ui> {
+    pub fn combo<'p>(&self, label : ImStr<'p>, current_item : &mut i32, items : &mut Vec<ImStr<'p>>, items_count : i32, height_in_items : i32) -> bool{
+        let items_inner : Vec<*const c_char> = items.into_iter().map(|item| item.as_ptr()).collect();
+        unsafe {
+            imgui_sys::igCombo(label.as_ptr(), current_item, items_inner.as_ptr() as *mut *const c_char, items_count, height_in_items)
+        }
+    }
+
+    pub fn combo2<'p>(&self, label : ImStr<'p>, current_item :&mut i32, items : ImStr<'p>, height_in_items : i32) -> bool {
```

Comment:
C-style strings do not support interior NULs, because there is no way to distinquish between them and the terminator NUL. In the Rust standard library, the [CString](https://doc.rust-lang.org/std/ffi/struct.CString.html) type has a guarantee that no interior NULs are used.

ImStr is basically CString with two differences:

* Guaranteed UTF-8
* Can be constructed compile-time without runtime cost using the im_str! macro

The interior NULs are not currently checked, but the semantics still apply. Like I said earlier, I might add checks for the semantics at some point, so I don't want code that intentionally breaks them.

Also, I've thought about the `combo2` function, and I don't see what the value of that function is. If we already have `combo`, why would we need another function? Just because the underlying library has it, doesn't mean the safe wrapper has to provide it.